pages/[lobby]/lobby.js

```diff
@@ -13,7 +13,9 @@ function Container() {
   const onInit = () => {
     const emit = loginData.create ? 'createLobby' : 'joinLobby';
     const payload = { name: loginData.name, lobby: loginData.lobby };
-    socket.emit(emit, payload);
+    if (payload.name && payload.lobby) {
```

Comment:
what if we do `if (!payload.name || !payload.lobby) { socket.disconnect(); return }`?